docs/howto/how-to-make-tensorflow-plugin-with-ui/tensorflow-plugin-with-ui.md

```diff
@@ -0,0 +1,1488 @@
+    })
+    let vectorzationSample = model.Configuration.datasetInfo.vectorizedSample
+    return (
+        <Container>
+        <Typography variant="h1"> Data Info </Typography>
+        <Grid alignItems="flex-end" container spacing={1}>
+            <Grid item> <DataTable rows={tableRows} /> </Grid>
+            <Grid item> <VectorizationPreview {...vectorzationSample} /> </Grid>
+            <Grid item> <TokenTranslation /></Grid>
+        </Grid>
+        </Container>)
+}
+```
+
+A data table that accepts `tableRows` to preview samples of our test data as it feeds into our model. Additionally, some vectorization sampling and token samples, as we've defined them in our backend.
+
+##### DataInputSection
+
+```jsx
+import React, {useContext, useState} from "react";
+
+import { ConfigTextInput } from './config-inputs';
+
+import { Container, Grid, Typography } from '@alteryx/ui';
+import { DS_CONFIG } from '../constants';
+
+import {Context as UiSdkContext} from "@alteryx/react-comms"
+import { DataInfo } from "./DataInfo";
+
+const DataInputSection = () => {
+    const [model, _] = useContext(UiSdkContext);
+    const {datasetInfo} = model.Configuration.datasetInfo
+    let inputItems = [
+        {configType:DS_CONFIG, vKey:"datasetTargetDir", elId:"dataset-target-dir", label:"Dataset Src Dir"},
+        {configType:DS_CONFIG, vKey:"trainingSetDir", elId:"dataset-train-dir", label:"Dataset Training Directory"},
+        {configType:DS_CONFIG, vKey:"testSetDir", elId:"dataset-test-dir", label:"Dataset Test Directory"},
+        {configType:DS_CONFIG, vKey:"batchSize", elId:"batch-size", inputType:"number", label:"Batch Size"},
+        {configType:DS_CONFIG, vKey:"seed", elId:"seed", inputType:"number", label:"seed"},
+    ]
+
+    const configTextInputs = inputItems.map((item) => {
+        let val = model.Configuration[item.configType][item.vKey]
+        
+        return <ConfigTextInput {...item} value={val} />
+    })
+
+    return (
+    <Container>
+        <Typography variant="h1"> Data Input </Typography>
+        <Grid alignItems="flex-end" container spacing={2}>
+            {configTextInputs.map((field, index) => {
+                return (<Grid item key={`ds-conf-grid-${index}`}>
+                    {field}
+                    </Grid>)
+            })}
+        </Grid>
+            <DataInfo  />
+        </Container>
+    )
+}
+
+export default DataInputSection;
+```
+
+We define the attributes and key-value pairs for several config text inputs using our earlier defined `ConfigTextInput`.
+Then, we bundle them into a grid to neatly display the inputs. You may have noticed, but note the `key` values and `labels and how they match our `provider.tool_config` values we mocked earlier.
+As we've discussed, these will be how the User (s) define their data options.
+There will also be similar additions for other values in the coming src.
```

Comment:
TW review:

We define the attributes and key-value pairs for several config text inputs using the earlier defined `ConfigTextInput`.
Then, we bundle them into a grid to neatly display the inputs. You might have already noticed, but note the `key` values and `labels and how they match the `provider.tool_config` values we mocked earlier. As we've discussed, this is how users define their data options. There are similar additions for other values in the coming src.